### PR TITLE
semantic: Harmonize headers

### LINKED ISSUE #212: settings: panel headers are too big as well as inconsistent among page.s
## Describe the bug

See the screenshots below and note the 

Screenshot 2020-11-29 at 02 23 30

The text "Sessions" should be smaller so that the two panels (settings and sessions) are the same size.

This is a general issues for all the settings pages.

Note also, that OAuth2Server settings pages are also different from the others.

Screenshot 2020-11-30 at 00 00 50
Screenshot 2020-11-30 at 00 00 46
Screenshot 2020-11-29 at 02 23 30

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes #212 

image

image

image

### CODE REVIEW

**Review by ppanero — COMMENTED** [review]
At eye measure, the text is now the same size. However, the headers are still not aligned:

Screenshot 2020-12-01 at 15 31 20

The right one is bigger than the left. I would bet is some padding on the icon. Alternatively (ask lars) make the left ones bigger.

**Comment by jrcastro2** [comment]
> At eye measure, the text is now the same size. However, the headers are still not aligned:
> 
> Screenshot 2020-12-01 at 15 31 20
> 
> The right one is bigger than the left. I would bet is some padding on the icon. Alternatively (ask lars) make the left ones bigger.

Thanks!
Fixed:
image

**Review by ppanero — APPROVED** [review]

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides [hunk 1]
```diff
@@ -34,6 +34,10 @@ li ~ .item{
   padding-left: 0em !important;
 }
 
+.right.floated.right.aligned.column {
```
Comment (ppanero) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:37]: quick comment: just wondering why this (and the other override) is not needed on the oauthclient PR.

Comment (jrcastro2) [reply invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:37]: Hmm in the `invenio-oauth2server` there are no css rules defined, as I understand we define them in `invenio-theme` and we get them from here

**Pull request merged**